tests/models/bridgetower/test_image_processing_bridgetower.py

```diff
@@ -39,7 +39,7 @@ def __init__(
         do_rescale: bool = True,
         rescale_factor: Union[int, float] = 1 / 255,
         do_normalize: bool = True,
-        do_center_crop: bool = True,
```

Comment:
Yes, was conflicted on this... this is the current version of `preprocess` for bridgetower in main, missing a new variable declaration:
https://github.com/huggingface/transformers/blob/115ac94d062b1950467ac3dd521e2cd504f626db/src/transformers/models/bridgetower/image_processing_bridgetower.py#L448
In that case, the `do_center_crop` that was used in `preprocess` was the `preprocess` default, i.e. `None` instead of whatever was in the `__init__`, right?

Reply:
@amyeroberts on that, bridgetower did default `do_center_crop`to being True, but `preprocess` was not capturing it (it was a bug). The `get_expected_values()` method does not include mentions on `center_crop` and will crate expected uncropped values. From that 
1) Either change `...expected_values...` getter to something that includes cropping in logic
2) change default of tester to match previous behaviour